pkg/scheduler/cache/dumper.go

```diff
@@ -0,0 +1,64 @@
+package cache
```

Comment:
please add the ut.

Reply:
I tried, but it just print some logs, maybe it's not necessory?